airbyte-integrations/connectors/destination-redshift/src/main/resources/spec.json

```diff
@@ -102,6 +102,14 @@
         "description": "The corresponding secret to the above access key id.",
         "title": "S3 Access Key",
         "airbyte_secret": true
+      },
+      "part_size": {
+        "type": "integer",
+        "minimum": 10,
+        "maximum": 100,
+        "examples": ["10"],
+        "description": "Files are streamed to S3 in parts. This determines the size of each part, in MBs. As S3 has a limit of 10,000 parts per file, part size affects the table size. This is 10MB by default, resulting in a default limit of 100GB tables. Increase this for larger tables. A larger part size will result in larger memory requirements. Only relevant for COPY.",
```

Comment:
can you add in the description that this otptional. the average user shouldn't need to worry about setting this. dunno if we should make it optional or set a default? but i guess it should also be clear what the default is.

Reply:
I'll make it optional. The default is 10MB and is handled in the Copier code. Will leave comments and all explaining this.